packages/hdwallet-gridplus/src/bitcoin.ts

```
@@ -0,0 +1,627 @@
+          throw new Error(`Input ${i} missing hex field (raw previous transaction)`);
+        }
+
+        const prevTx = bitcoin.Transaction.fromHex(input.hex);
+        const prevOutput = prevTx.outs[input.vout];
+        const scriptPubKey = prevOutput.script;
+
+        // Detect input type from scriptPubKey
+        // P2WPKH (SegWit): 0x00 0x14 <20-byte-hash>
+        const isSegwit = scriptPubKey.length === 22 &&
+                         scriptPubKey[0] === 0x00 &&
+                         scriptPubKey[1] === 0x14;
```

ESLint

[error] Replace `⏎·························scriptPubKey[0]·===·0x00·&&⏎························` with `·scriptPubKey[0]·===·0x00·&&`

(prettier/prettier)